Tensile/KernelWriterAssembly.py

```diff
@@ -3763,6 +3763,31 @@ def openPersistentLoop(self, kernel):
     if kernel["PersistentKernel"] or kernel["StreamK"]:
       if kernel["StreamK"]:
         # Workload calculations
+        if kernel["StreamKXCCMapping"]:
+          sXCC = self.sgprPool.checkOut(1, "XCC", preventOverflow=0)
+          sGridC = self.sgprPool.checkOut(1, "sGridC", preventOverflow=0)
+          sGridF = self.sgprPool.checkOut(1, "sGridF", preventOverflow=0)
+          sGridM = self.sgprPool.checkOut(1, "sGridM", preventOverflow=0)
+          kStr += inst("s_add_u32", sgpr(sGridC), sgpr("skGrid"), 7, "ceil(grid/8)")
+          kStr += inst("s_lshr_b32", sgpr(sGridC), sgpr(sGridC), 3, "ceil(grid/8)")
+          kStr += inst("s_lshr_b32", sgpr(sGridF), sgpr("skGrid"), 3, "floor(grid/8)")
+          kStr += inst("s_lshl_b32", sgpr(sGridM), sgpr(sGridF), 3, "mod(grid,8)")
+          kStr += inst("s_sub_u32", sgpr(sGridM), sgpr("skGrid"), sgpr(sGridM), "mod(grid,8)")
```

Comment:
We do not need 2 instructions to calculate mod8.
Simply
  sGridM = skGrid & 7

Reply:
Updated version fixes this